docs/source/resources/define-metadata.rst

```diff
@@ -1311,6 +1312,24 @@ contents.
          - *.some-extension
          - somefolder/*.some-extension
 
+Files can be excluded by specifying `files` as a dictionary separating
+files to `include` from those to `exclude`. Files installed to the prefix
+by host dependencies are automatically excluded when the include/exclude
+syntax is used:
```

Comment:
We need to mention the precedence here, i.e., `exclude` entries have higher priority as per the implementation.

Reply:
This mentions the new behavior, i.e., "Files [...] excluded when the include/exclude syntax is used".
But since this is a deviation from the previous "include" logic (which would be more of a `force-include` since it allows packaging pre-existing files.), I think we should make the difference more prevalent/explain the behavior for the old non-dict case more explicitly.

Reply:
> We need to mention the precedence here, i.e., `exclude` entries have higher priority as per the implementation.

`exclude` has a higher priority than which operation? It's unclear to me what ambiguity we would be removing by mentioning priority.

> This mentions the new behavior, i.e., "Files [...] excluded when the include/exclude syntax is used". But since this is a deviation from the previous "include" logic (which would be more of a `force-include` since it allows packaging pre-existing files.), I think we should make the difference more prevalent/explain the behavior for the old non-dict case more explicitly.

```suggestion
When using `outputs/files`, any file in the prefix (including those
installed by host dependencies) matching one of the glob expressions is
included in the output. Greater control over file matching may be
achieved by using `outputs/files/include` and `outputs/files/exclude`
glob expressions. When using include/exclude, only files installed by
the current recipe are considered. i.e. files in the prefix installed
by host dependencies are not matched.
```

Reply:
> > We need to mention the precedence here, i.e., `exclude` entries have higher priority as per the implementation.
> 
> `exclude` has a higher priority than which operation?

The kind of precedence where if you do
```
  files:
    include:
      - lib/libfoo.so
    exclude:
      - lib/libfoo.so
```
the exclusion wins.

Reply:
It is now explained that when files match both exclude and include, they are excluded. d1b12a16537cb7507bcf9ba8e5984c3c819bdac2